package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "clean-dist": "rm -rf packages/*/dist && rm -rf packages/*/tsconfig.tsbuildinfo",
-    "test": "prettier --check . && eslint . --max-warnings 6 && yarn build && yarn workspace flipper-plugin-android-performance-profiler test && jest",
+    "test": "prettier --check . && yarn build && eslint . --max-warnings 6 && yarn workspace flipper-plugin-android-performance-profiler test && jest",
```

Comment:
this is because there's no `@perf-profiler/e2e/index.ts` anymore so eslint fails to resolve the import if the JS isn't built